src/ranged.cpp

```diff
@@ -948,43 +943,42 @@ static void do_aim( player *p, std::vector <Creature *> &t, int &target,
         p->recoil = std::max(MIN_RECOIL, p->recoil);
     }
 
-    const int aim_amount = p->aim_per_time( *relevant, p->recoil );
+    const double aim_amount = p->aim_per_move( *relevant, p->recoil );
     if( aim_amount > 0 ) {
         // Increase aim at the cost of moves
-        p->moves -= 10;
+        p->moves--;
```

Comment:
Aiming is either in 100 move blocks via pause (`.`) key or in actual units via specific mode (`m`, `p`, `c`). In the case of `m` you even get a display of move cost which I probably should add for `p` and `c`